Src/Projects/constraint_twistextraction/constraint_twistextraction.cxx

```diff
@@ -0,0 +1,306 @@
+/**	\file	constraint_twistextraction.cxx
+
+Mikkel Brons-Frandsen 2021
+
+GitHub page - https://github.com/Neill3d/OpenMoBu
+Licensed under The "New" BSD License - https://github.com/Neill3d/OpenMoBu/blob/master/LICENSE
+
+*/
+
+//--- Class declarations
+#include "constraint_twistextraction.h"
+
+#include <vector>
+
+//--- Registration defines
+#define	CCONSTRAINTTWISTEXTRACTION__CLASS CCONSTRAINTTWISTEXTRACTION__CLASSNAME
+#define CCONSTRAINTTWISTEXTRACTION__NAME "Twist Extraction Constraint"
+#define CCONSTRAINTTWISTEXTRACTION__LABEL "Twist Extraction Constraint"
```

Comment:
I would suggest to call it just Twist Extraction, as it will be shown in the constraints section already